Sentiment_analysis/.env

```diff
@@ -0,0 +1 @@
+SECRET_KEY = "g%sfzyu*_m719@hk7qh1hqg-e+!9y6hw2x5)aq1q)4t%hk!!+t"
```

Comment:
The secret key still inside the repository.

Reply:
But if delete it, no one will be able to run the app...May I be mistaken?

Reply:
Just make it look not like a secret that you accidentally put into the repo.  Show your intention, you know that it is bad, but risk was calculated.